extensions/blocks/premium-content/_inc/tabs.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
I can't figure out where this is currently being used. Am I missing something obvious here? :\

Reply:
Nope, looks like it is no longer used with previous UI updates.